src/main/drivers/bus_busdev_i2c.c

```diff
@@ -49,4 +49,11 @@ bool i2cBusReadRegister(const busDevice_t * dev, uint8_t reg, uint8_t * data)
     const bool allowRawAccess = (dev->flags & DEVFLAGS_USE_RAW_REGISTERS);
     return i2cRead(dev->busdev.i2c.i2cBus, dev->busdev.i2c.address, reg, 1, data, allowRawAccess);
 }
+bool i2cBusBusy(const busDevice_t *dev, bool *error)
```

Comment:
This is breaking existing targets.

You can either turn this function into a macro that translate to false (better code footprint for other targets) or mark the parameters with UNUSED()

Reply:
yelp ,i2cBusBusy function 's unused parm "dev“ and "error "  may cause a unused param warning , can be fix in the future

Reply:
The github builds enable warnings as errors. It would save time if you could build like that locally as well.

Try running cmake -DWARNINGS_AS_ERRORS=ON and running make release to build all targets as well.

Reply:
@shanggl
After tests on real AT32 FC there are few things I noticed and want to share (all for NEUTRONRCF435AIO target):
- Gyro orientation for BMI270 should be CW180_DEG
- ADC readings is wrong, both Voltage and Current (I don't know why, have no time to dig)
- LED strip not working (as I see it causes some conflicts with UART3 TX)
- Rename UART8 -> UART5, as on FC pads are R5 and T5 (at least on one I have)

Configurator build able to flash this MCU attached (if someone need it)

Reply:
> The github builds enable warnings as errors. It would save time if you could build like that locally as well.
> 
> Try running cmake -DWARNINGS_AS_ERRORS=ON and running make release to build all targets as well.

All warnings has been fix yesterday , and tested the STM32F411 F405 F722 and  H743 , all targets 0 errors  0 warnings